### DISCUSSION TITLE
The solution is grouped into one of the invalid solutions right after being submitted

### DISCUSSION THREAD

**Discussion by Yang-Wei-Ting** [codewars/codewars.com#2627] [General]
Hi code warriors.

Right after I submitted my solution for [this kata](https://www.codewars.com/kata/51675d17e0c1bed195000001/python), it is grouped into one of the invalid solutions.
I checked the kata revision history and didn't see any recent test update.
Later, I submitted the same solution again, it still passed all tests.

Now I wonder is my solution valid or invalid?

**Comment by Blind4Basics** [comment]
with 22000 solutions.... 'would be helopful that you drop the direct link to that solution... XD

**Comment by Yang-Wei-Ting** [comment]
My bad. Didn't know the direct link exists.
[Here](https://www.codewars.com/kata/reviews/55a68d551dc0561dc9000074/groups/55eae84e1f4a693adf000009) you are.

**Comment by Blind4Basics** [comment]
Surely it has been invalidated at some point in the past, because of a change in the test suite and it has never been revalidated. Done.

**Reply by Yang-Wei-Ting** [reply]
Thanks!
